### DISCUSSION TITLE
Front Page

### DISCUSSION THREAD

**Discussion by petrasvestartas** [alshedivat/al-folio#736] [Q&A]
Hi,

How can change the front page link?
For example, when I open the browser I would like to see **projects page i**nstead of **about section**?

**Comment by rohandebsarkar** [comment]
Change the permalink in the page YAML to `/`:
https://github.com/alshedivat/al-folio/blob/032f0e62bafa79fe4037adf6a2acfc016dc2db02/_pages/projects.md?plain=1#L4

**Reply by rohandebsarkar** [reply]
Also make sure to delete the about page or change its permalink to something else.
